### PR TITLE
Fix/msg system config and banner

### LINKED ISSUE #3874: Message System config: WebUSB API not working on macOS in Google Chrome 91
**Describe the bug**
WebUSB API is not working on macOS in Google Chrome 91 so users are not able to use the application without Bridge installed.

**Steps to reproduce:**
1. Have Google Chrome 91
2. Launch the app
3. Connect the device

**Expected result:**
Device is connected.

**Actual result:**
Device cannot be connected.

**Screenshots:**
https://user-images.githubusercontent.com/33235762/121348727-fae33080-c928-11eb-8dc9-e39d2e35a560.mov

### PR DESCRIPTION
**Pull request by tomasklim**
- update the wording of WebUSB Chrome 91 bug and remove cta
- display the message only in Suite which has msg system banners on top feature implemented
- fix cancel button position when cta button was missing (centred vertically)

### CODE REVIEW

**Comment by tomasklim** [comment]
Screenshot 2021-06-11 at 14 44 14
Screenshot 2021-06-11 at 14 46 21

File: packages/suite-data/src/message-system/config/config.v1.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
                     "environment": {
                         "desktop": "!",
                         "mobile": "!",
-                        "web": "*"
+                        "web": ">21.6.2"
```
Comment (tsusanka) [inline packages/suite-data/src/message-system/config/config.v1.json]: Shouldn't this be `>=21.6.2`?

Comment (tomasklim) [reply packages/suite-data/src/message-system/config/config.v1.json]: We have `21.7.0` in `package.json`.

**Review by goodhoko — APPROVED** [review]
Seems Legit.

File: packages/suite-data/src/message-system/config/config.v1.json (hunk 1 above)
Comment (tomasklim) [reply packages/suite-data/src/message-system/config/config.v1.json]: So it depends if you plan to change it to `21.6.2`

Comment (tomasklim) [reply packages/suite-data/src/message-system/config/config.v1.json]: I am changing it to `>21.6.1` and it will solve both problems

Comment (tsusanka) [reply packages/suite-data/src/message-system/config/config.v1.json]: Yes, `21.7.0` denotes August (beta) release. But if we plan to release in June one more time (which we do) the release will be versioned as `21.6.2`. And we'll change that in the release branch. Described also [in the docs](https://docs.trezor.io/trezor-suite/releases/versioning.html).

Comment (tomasklim) [reply packages/suite-data/src/message-system/config/config.v1.json]: Fixed and squashed

**Review by tsusanka — APPROVED** [review]
All good. Nit: please do not resolve comments on your own next time, the one that writes the comment should do the final "resolve".

**Comment by bosomt** [comment]
**QA OK**


https://github.com/trezor/trezor-suite/issues/3874#issuecomment-862312514

**Info:**
 - Suite version: desktop 21.7.0 (6796f451738b4cd59ecd9efb3257743bbe8398a1)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 11_4_0) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/21.7.0 Chrome/89.0.4389.69 Electron/12.0.0 Safari/537.36
 - OS: MacIntel
 - Screen: 1440x900
 - Device: model T 2.4.0 regular

**Pull request merged**